scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -251,21 +249,21 @@ sealed trait TypedPipe[+T] extends Serializable {
    *
    * Same as groupAll.aggregate.values
    */
-  def aggregate[B, C](agg: Aggregator[T, B, C]): ValuePipe[C] =
+  def aggregate[B, C](agg: Aggregator[T, B, C])(implicit m: Quoted): ValuePipe[C] =
     ComputedValue(groupAll.aggregate(agg).values)
 
   /**
    * Put the items in this into the keys, and unit as the value in a Group
    * in some sense, this is the dual of groupAll
    */
   @annotation.implicitNotFound(msg = "For asKeys method to work, the type in TypedPipe must have an Ordering.")
-  def asKeys[U >: T](implicit ord: Ordering[U]): Grouped[U, Unit] =
+  def asKeys[U >: T](implicit ord: Ordering[U], m: Quoted): Grouped[U, Unit] =
     map(WithConstant(())).group
 
   /**
    * If T <:< U, then this is safe to treat as TypedPipe[U] due to covariance
    */
-  protected def raiseTo[U](implicit ev: T <:< U): TypedPipe[U] =
+  protected def raiseTo[U](implicit ev: T <:< U, m: Quoted): TypedPipe[U] =
```

Comment:
I don't think we are using `m` here are we? This is really just a typesafe cast.

Reply:
removed